docs/production-deployment/cloud/service-health.mdx

```diff
@@ -0,0 +1,90 @@
+---
+id: service-health
+title: Service health
+sidebar_label: Service health
+description: Use Temporal Cloud metrics to monitor your production deployment Service health.
+toc_max_heading_level: 4
+tags:
+  - temporal-cloud
+  - metrics
+  - observability
+  - health
+---
+
+Temporal Cloud metrics help you to monitor the Service health of your production deployments.
+
+## Monitor availability issues
+
+When you see a sudden drop in Worker resource utilization, verify whether Temporal Cloud's API is showing an increased error rate.
+
+### Reference Metrics
+
+- [temporal\_cloud\_v0\_service\_latency\_bucket](/production-deployment/cloud/metrics/reference#temporal_cloud_v0_service_latency_bucket)
+
+### Prometheus Query
+
+P99 service lag (histogram):
+
+```
+histogram_quantile(0.99, sum(rate(temporal_cloud_v0_service_latency_bucket[5m])) by (temporal_namespace, operation, le))
```

Comment:
With the release of Grafana v7.2 (Sep 2020), the recommended interval in the query is `$__rate_interval`: https://grafana.com/docs/grafana/latest/datasources/prometheus/template-variables/#use-__rate_interval

I noticed we are using `5m` and `1m` in this doc that we could consider to align to Grafana's recommendation.